dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/Main.java

```diff
@@ -35,17 +37,20 @@ public void start(final Stage primaryStage) throws Exception {
         primaryStage.setMinWidth(Constants.MIN_WIDTH);
         final SettingsStore settings = SettingsStore.loadSettingsStore();
         // Create the log service before any logging has started..
-        final LogService logService = new LogService(settings.getLogSettings());
+        final ApplicationLoggerSettings applicationLoggerSettings = new ApplicationLoggerSettings(settings.getLogSettings());
+        applicationLoggerSettings.restoreLoggingSettings();
         final SavedJobPrioritiesStore savedJobPrioritiesStore = SavedJobPrioritiesStore.loadSavedJobPriorties();
         final JobInterruptionStore jobInterruptionStore = JobInterruptionStore.loadJobIds();
         final SavedSessionStore savedSessionStore = SavedSessionStore.loadSavedSessionStore();
         final ResourceBundle resourceBundle = ResourceBundle.getBundle("lang", new Locale(ConfigProperties.getInstance().getLanguage()));
         final DataFormat dataFormat = new DataFormat(resourceBundle.getString("dataFormat"));
         final JobWorkers jobWorkers = new JobWorkers(settings.getProcessSettings().getMaximumNumberOfParallelThreads());
         final Logger injectorLogger = LoggerFactory.getLogger("Injector");
-        LOG.info("Starting Deep Storage Browser v2.0");
+        final LoggingServiceImpl loggingService = new LoggingServiceImpl();
+        LOG.info("Starting Deep Storage Browser v2.1");
```

Comment:
I actually have that ready to go in a separate PR
https://github.com/SpectraLogic/ds3_java_browser/pull/32